qa/L0_grpc/test.sh

```diff
@@ -65,6 +65,8 @@ if [[ "$(< /proc/sys/kernel/osrelease)" == *microsoft* ]]; then
     SIMPLE_CUDASHM_CLIENT_PY=${SDKDIR}/python/simple_grpc_cudashm_client.py
     SIMPLE_MODEL_CONTROL_PY=${SDKDIR}/python/simple_grpc_model_control.py
     SIMPLE_REUSE_INFER_OBJECTS_CLIENT_PY=${SDKDIR}/python/reuse_infer_objects_client.py
+    SIMPLE_KEEPALIVE_CLIENT_PY=${SDKDIR}/python/simple_grpc_keepalive_client.py
```

Comment:
See my request to convert these examples to unit tests in the client PR.

Reply:
As we discussed - I think given the generic nature of the channel_args constructor, these will be a good example to keep for reference when users need custom arguments not exposed by Triton. Moving forward, I will be more picky whether new features should be an example or a unit test.